src/main/java/org/opentripplanner/updater/spi/ResultLogger.java

```diff
@@ -34,16 +36,16 @@ public static void logUpdateResult(String feedId, String type, UpdateResult upda
           var value = errorIndex.get(key);
           var tripIds = value.stream().map(UpdateError::debugId).collect(Collectors.toSet());
           LOG.error(
-            "[feedId: {}, type={}] {} failures of type {}: {}",
-            feedId,
-            type,
+            "[{} {}] {} failures of {}: {}",
```

Comment:
This should be logged as a warning, not en error. There is nothing wrong in OTP, it is the data OTP receivs that is wrong.

Reply:
Done in 6f1ead0357f08998eabcc9097f2ae88961f139b1